examples/vortex-mpi.py

```diff
@@ -52,20 +53,41 @@
 from mirgecom.initializers import Vortex2D
 from mirgecom.eos import IdealSingleGas
 
+from logpyle import (LogManager, IntervalTimer, add_general_quantities,
+        add_simulation_quantities, add_run_info)
+
+from mirgecom.logging_quantities import (DependentQuantity, ConservedQuantity,
+                                         KernelProfile, add_package_versions)
+
 
 logger = logging.getLogger(__name__)
 
 
 @mpi_entry_point
-def main(ctx_factory=cl.create_some_context):
+def main(ctx_factory=cl.create_some_context, use_profiling=False, use_logmgr=False):
```

Comment:
Same suggested changes as in `sod-mpi.py`.

We can't keep repeating this stuff. Could you see about factoring out more parts of this into common functions?

Reply:
I have adapted the changes from sod-mpi to here, and restored the master version of sod-mpi for now until we have the API nailed down. 

- removed argument passing, replaced it with a bool config value in the driver
- passed the communicator